annif/corpus/subject.py

```diff
@@ -81,6 +91,12 @@ def labels_to_uris(self, labels):
                 for subject_id in (self.by_label(label) for label in labels)
                 if subject_id is not None]
 
+    def empty_labels(self):
```

Comment:
I'm not sure I like the method name since it's so low level and doesn't even return labels. How about `deprecated_ids`?